common/source/docs/common-smart-battery-landingpage.rst

```diff
@@ -0,0 +1,43 @@
+.. _common-smartbattery-landingpage:
+
+===============
+Smart Batteries
+===============
+
+ArduPilot supports two types of "smart batteries" that communicate using `SMBus <https://en.wikipedia.org/wiki/System_Management_Bus>`__:
+
+- the `Rotoye BatMon <https://rotoye.com/>`__
```

Comment:
I know we would like to promote new hardware, but we have to balance against confusing the majority of users which only use Stable....that is why anything not supported in stable is mentioned only in the Upcoming Features (common-master-features) and not in the main wiki....and adding it to the main with version labels becomes a nightmare several version s down the road....so, you have it already linked on the Upcoming features page, but should not be listed here until 4.1 is released...sorry..

Reply:
If so, what are the recommended changes?

Reply:
just remove the royote line